supplements/astral-adventurers-guide/items.xml

```diff
@@ -0,0 +1,682 @@
+			<set name="rarity">Very Rare</set>
+			<set name="attunement">false</set>
+		</setters>
+	</element>
+
+	<element name="Spelljamming Helm" type="Magic Item" source="Astral Adventurer's Guide" id="ID_WOTC_AAG_MAGIC_ITEM_SPELLJAMMING_HELM">
+		<description>
+			<p>The function of this ornate chair is to propel and maneuver a ship on which it has been installed through space and air. It can also propel and maneuver a ship on water or underwater, provided the ship is built for such travel. The ship in question must weigh 1 ton or more.</p>
+			<d class="indent">The sensation of being attuned to a <i>spelljamming helm</i> is akin to the pins-and-needles effect one experiences after one’s arm or leg falls asleep, but not as painful.</d>
+			<d class="indent">While attuned to a <i>spelljamming helm</i> and sitting in it, you gain the following abilities for as long as you maintain concentration (as if concentrating on a spell):</d>
+			<ul>
+				<li>You can use the <i>spelljamming helm</i> to move the ship through space, air, or water up to the ship’s speed. If the ship is in space and no other objects weighing 1 ton or more are within 1 mile of it, you can use the <i>spelljamming helm</i> to move the vessel fast enough to travel 100 million miles in 24 hours.</li>
+				<li>You can steer the vessel, albeit in a somewhat clumsy fashion, in much the way that a rudder or oars can be used to maneuver a seafaring ship.</li>
+				<li>At any time, you can see and hear what’s happening on and around the vessel as though you were standing in a location of your choice aboard it.</li>
+			</ul>
+			<d class="indent"><b><i>Transfer Attunement.</i></b> You can use an action to touch a willing spellcaster. That creature attunes to the <i>spelljamming helm</i> immediately, and your attunement to it ends.</d>
+			<div class="sidebar">
+				<h5>COST OF A SPELLJAMMING HELM</h5>
+				<p>A <i>spelljamming helm</i> propels and steers a ship much as sails, oars, and rudders work on a seafaring vessel, and a <i>spelljamming helm</i> is easy to create if one has the proper spell. <i>Create spelljamming helm</i> spell has a material component cost of 5,000 gp, so that's the least one can pay to acquire a <i>spelljamming helm</i>.</p>
+				<d class="indent">Wildspace merchants, including <b>dohwars</b> and <b>mercanes</b> (both described in <i>Boo's Astral Menagerie</i>, on pages 19 and 37 respectively), typically sell a spelljammer helm for substantially more than it cost to make. How much more depends on the market, but 7,500 gp would be a reasonable demand. A desperate buyer in a seller's market might pay 10,000 gp or more.</d>
+			</div>
+		</description>
+		<setters>
+			<set name="keywords">spelljammer, spellcaster</set>
+			<set name="category">Wondrous Items</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Rare</set>
+			<set name="attunement" addition="Requires Attunement by a Spellcaster">true</set>
+		</setters>
+	</element>
```

Comment:
Not sure what <d> is about but it screwed with the formatting
```suggestion
	<element name="Spelljamming Helm" type="Magic Item" source="Astral Adventurer's Guide" id="ID_WOTC_AAG_MAGIC_ITEM_SPELLJAMMING_HELM">
		<description>
			<p>The function of this ornate chair is to propel and maneuver a ship on which it has been installed through space and air. It can also propel and maneuver a ship on water or underwater, provided the ship is built for such travel. The ship in question must weigh 1 ton or more.</p>
			<p class="indent">The sensation of being attuned to a <i>spelljamming helm</i> is akin to the pins-and-needles effect one experiences after one’s arm or leg falls asleep, but not as painful.</p>
			<p class="indent">While attuned to a <i>spelljamming helm</i> and sitting in it, you gain the following abilities for as long as you maintain concentration (as if concentrating on a spell):</p>
			<ul>
				<li>You can use the <i>spelljamming helm</i> to move the ship through space, air, or water up to the ship’s speed. If the ship is in space and no other objects weighing 1 ton or more are within 1 mile of it, you can use the <i>spelljamming helm</i> to move the vessel fast enough to travel 100 million miles in 24 hours.</li>
				<li>You can steer the vessel, albeit in a somewhat clumsy fashion, in much the way that a rudder or oars can be used to maneuver a seafaring ship.</li>
				<li>At any time, you can see and hear what’s happening on and around the vessel as though you were standing in a location of your choice aboard it.</li>
			</ul>
			<p class="indent"><b><i>Transfer Attunement.</i></b> You can use an action to touch a willing spellcaster. That creature attunes to the <i>spelljamming helm</i> immediately, and your attunement to it ends.</p>
			<div class="sidebar">
				<h5>COST OF A SPELLJAMMING HELM</h5>
				<p>A <i>spelljamming helm</i> propels and steers a ship much as sails, oars, and rudders work on a seafaring vessel, and a <i>spelljamming helm</i> is easy to create if one has the proper spell. <i>Create spelljamming helm</i> spell has a material component cost of 5,000 gp, so that's the least one can pay to acquire a <i>spelljamming helm</i>.</p>
				<p class="indent">Wildspace merchants, including <b>dohwars</b> and <b>mercanes</b> (both described in <i>Boo's Astral Menagerie</i>, on pages 19 and 37 respectively), typically sell a spelljammer helm for substantially more than it cost to make. How much more depends on the market, but 7,500 gp would be a reasonable demand. A desperate buyer in a seller's market might pay 10,000 gp or more.</p>
			</div>
		</description>
		<setters>
			<set name="keywords">spelljammer, spellcaster</set>
			<set name="category">Wondrous Items</set>
			<set name="type">Wondrous Item</set>
			<set name="rarity">Rare</set>
			<set name="attunement" addition="Requires Attunement by a Spellcaster">true</set>
		</setters>
	</element>
```